python/feathub/feature_views/sql_feature_view.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2022 The Feathub Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Dict, Sequence, Optional, List
+
+from feathub.common.exceptions import FeathubException
+from feathub.feature_views.feature import Feature
+from feathub.feature_views.feature_view import FeatureView
+from feathub.registries.registry import Registry
+from feathub.table.schema import Schema
+from feathub.table.table_descriptor import TableDescriptor
+
+
+class SqlFeatureView(FeatureView):
+    """
+    Acquires features by evaluating the given SQL query statement using a specific
+    Processor.
+
+    Apart from the following requirements, The SQL statement may use any sql
```

Comment:
The doc seems to involve too much implementation detail.

How about using the following doc:

```
"""
Derives features by evaluating a given SQL statement.

Currently, the SQL statement is executed by the underlying processor's SQL interface
without any change and therefore its semantics depends on the processor used during
deployment.

While executing the same SQL statement on different processors usually yields
identical results, there is no guarantee of this. We plan to make SqlFeatureView
processor-agnostic in the future to ensure consistent semantics regardless of
processor choice.

NOTE: the tables used in the SQL statement should be built with the
FeatHubClient#build_features() before executing the SqlFeatureView.
"""
```